Sources/GRPCCodeGen/Internal/Translator/IDLToStructuredSwiftTranslator.swift

```diff
@@ -72,13 +72,23 @@ struct IDLToStructuredSwiftTranslator: Translator {
       }
     }
 
-    let fileDescription = FileDescription(
-      topComment: .preFormatted(codeGenerationRequest.leadingTrivia),
-      imports: try self.makeImports(
+    let imports: [ImportDescription]
+    if codeBlocks.isEmpty {
```

Comment:
Can we check this instead? It's easy to imagine unconditionally adding a code block which would result in this check failing.

```suggestion
    if codeGenerationRequest.services.isEmpty {
```